src/OrchardCore.Modules/OrchardCore.Media/Razor/OrchardRazorHelperExtensions.cs

```diff
@@ -28,6 +32,33 @@ public static string AssetUrl(this IOrchardHelper orchardHelper, string assetPat
 
             resizedUrl = fileVersionProvider.AddFileVersionToPath(orchardHelper.HttpContext.Request.PathBase, resizedUrl);
         }
+
+        return resizedUrl;
+    }
+
+    /// <summary>
+    /// Returns the relative URL of the specified asset path for a media profile with optional resizing parameters.
+    /// </summary>
+    public static async Task<string> AssetProfileUrlAsync(this IOrchardHelper orchardHelper, string assetPath, string imageProfile, int? width = null, int? height = null, ResizeMode resizeMode = ResizeMode.Undefined, bool appendVersion = false, int? quality = null, Format format = Format.Undefined)
```

Comment:
To make this non breaking, and async, I have made another helper called `AssetProfileUrlAsync`, which goes alongside the `AssetUrl` helper.

Otherwise it breaks the `AssetUrl` helper by making it `AssetUrlAsync`